nbs/00_utils.ipynb

```diff
@@ -12,17 +12,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        
</pre><p>Write documentation for this function. See <a href="https://birkhoffg.github.io/jax-relax/tutorials/contribution.html#write-documentations-in-jupyter-notebook" target="_blank">jax-relax - Contribute (birkhoffg.github.io)</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>